cg/services/flow_cell_service/flow_cell_service.py

```diff
@@ -0,0 +1,12 @@
+from cg.services.flow_cell_service.dtos import SequencingMetrics
+from cg.services.flow_cell_service.utils import create_metrics_dto
+from cg.store.store import Store
+
+
+class FlowCellService:
```

Comment:
```suggestion
class IlluminaFlowCellService:
```

We should avoid introducing generic class names for "FlowCell" at this point. It is specific for illumina flow cells here. We started this separation in the post processing flow to avoid mixing run instrument services accidentally.

Reply:
What should it be called instead?

Reply:
Changed naming https://github.com/Clinical-Genomics/cg/pull/3549/commits/002147338af541d15080e57049fcf424931ced07